RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -1204,6 +1204,63 @@ void auto_printNightcap()
 	auto_log_info("Nightcap is: " + to_pretty_string(auto_bestNightcap()), "blue");
 }
 
+void auto_overdrinkGreenBeers()
+{
+	//called after nightcap, auto_drinkNightcap() needs to have already made the necessary checks
+	if(!contains_text(holiday(),"St\. Sneaky Pete's Day") || !canDrink($item[green beer], false))
+	{
+		return;
+	}
+	familiar start_fam = my_familiar();
+	if(auto_have_familiar($familiar[Stooper]) //drinking does not break 100fam runs so do not use canChangeToFamiliar
```

Comment:
it's something inside a function of the bedtime file, not sure about making it available outside of bedtime since it's a check for drinking only that ignores 100%familiar and shouldn't be used to finish a run with the stooper or something like that